charts/opentelemetry-demo/values.yaml

```diff
@@ -392,6 +445,14 @@ components:
       env: true
     service:
       port: 8080
+    podAnnotations:
+      resource.opentelemetry.io/service.name: frontend-proxy
+      resource.opentelemetry.io/service.namespace: opentelemetry-demo
+      io.opentelemetry.discovery.metrics/enabled: 'true'
+      io.opentelemetry.discovery.metrics/scraper: httpcheck
+      io.opentelemetry.discovery.metrics/config: |
+          targets:
+            - endpoint: "http://`endpoint`"  # FIXME why isn't `endpoint` interpolated?
```

Comment:
@rogercoll could you please help me understand why the receiver creator doesn't extrapolate `` `endpoint` `` to create the `httpcheck` receiver when it successfully extrapolates the same `` `endpoint` `` for the NGinx receiver below?